docs/contributing/HACKING.md

```diff
@@ -52,7 +52,14 @@ You'll use this command a lot.
 
 ### Building
 
-In the base directory, run `cabal build`
+In the base directory, run `cabal build`.  if you want to build the full binary, run `./vendor_download.sh` first (requires a FOSSA-org `GITHUB_TOKEN` set in the environment).
+
+#### Note for Non-FOSSA users
+
+Currently, building the full-featured binary requires the use of external binaries fetched from closed-source repos in the FOSSA organization (`fossas`).
+If you build without these binaries, a compiler warning will be emitted.  If ghc's `-Werror` option is enabled, this will fail the build, and is therefore not enabled by default.
+
+There is no supported way for Non-FOSSA users to obtain these binaries, though we are working on a solution to allow this in the future.  As a result, **we cannot accept pull requests from any forked repository**, since those builds will fail in CI.
```

Comment:
Wait, so if they can't fork and PR, how are they supposed to contribute?  They can't create a branch on this repo, can they?  Maybe a note here about how we actually expect people to contribute, even if it's just a link to another section.

Reply:
They can't contribute.  That's the really gross part about this, we can't accept PR's from anyone outside the org at all due to this issue.